src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/iOS/PopupRenderer.ios.cs

```diff
@@ -135,17 +141,26 @@ void SetViewController()
 		{
 			IVisualElementRenderer currentPageRenderer;
 			var modalStackCount = Application.Current.MainPage?.Navigation?.ModalStack?.Count ?? 0;
-			var mainPage = Application.Current.MainPage;
+			var page = Application.Current.MainPage;
 			if (modalStackCount > 0)
 			{
 				var index = modalStackCount - 1;
-				currentPageRenderer = Platform.GetRenderer(mainPage!.Navigation!.ModalStack![index]);
+				page = page!.Navigation!.ModalStack![index];
```

Comment:
Let's avoid using the Null-Forgiving Operator, `!`. 

It's a code smell that'll just lead to future bugs.